src/Sulu/Bundle/AdminBundle/Resources/js/components/Toolbar/Items.js

```diff
@@ -0,0 +1,28 @@
+// @flow
+import type {ChildrenArray, Element} from 'react';
+import React from 'react';
+import Button from './Button';
+import Dropdown from './Dropdown';
+import Select from './Select';
+import toolbarStyles from './toolbar.scss';
+
+type Children = Element<typeof Button> | Element<typeof Dropdown> | Element<typeof Select>
```

Comment:
I think we've said that we only have the `Props` type in the component itself, and move the other stuff into a separate `types.js` file.

Reply:
should I create a `ItemsChildren type there?`

Reply:
Yeah, would make sense to me.